.github/scripts/randcurl.pl

```diff
@@ -76,10 +79,11 @@ sub addarg {
 }
 
 sub randarg {
-    my $nice = "abcdefhijklmnopqrstuvwqxyz".
-        "ABCDEFHIJKLMNOPQRSTUVWQXYZ".
-        "0123456789".
-        ",-?#$%!@ ";
+    my $nice =
+        "abcdefhijklmnopqrstuvwqxyz"
+      . "ABCDEFHIJKLMNOPQRSTUVWQXYZ"
+      . "0123456789"
+      . ",-?#$%!@ ";
```

Comment:
I'd strongly favor the existing style here and leave those at the end of the previous line. Same for `&&` and `||`.